src/geventhttpclient/client.py

```diff
@@ -211,24 +255,50 @@ def request(self, method, request_uri, body=b"", headers={}):
                 response._sent_request = request
                 return response
 
-    def get(self, request_uri, headers={}):
+    def get(self, request_uri, headers=None):
+        if headers is None:
```

Comment:
As the None handling is perfectly covered in `request`, I'd leave out all this redundant code here.

Reply:
Default argument value is mutable.
you could read this: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
I'm all for changing the default argument to None. It's just the `if`-clause that can be omitted, as None gets converted to a dict anyways in the `request` method.

Reply:
ok, excuse me..